packages/TextField/TextField.vue

```diff
@@ -1,29 +1,47 @@
 <template>
   <label>
     {{label}}
-    <input
-      :type="type"
-      :placeholder="placeholder"
-      :value="value"
-      @input="$emit(`input`, $event.target.value)"
-    />
+    <div class="password">
+      <template v-if="type==='password'">
+        <button @click="showPwd" :class="{actived: showPassword}">
+          <i v-if="showPassword" class="fas fa-eye"></i>
+          <i v-else-if="!showPassword" class="fas fa-eye-slash"></i>
+        </button>
+      </template>
+      <input :type="textType" :placeholder="placeholder" :value="value" @input="$emit(`input`, $event.target.value)" />
+    </div>
   </label>
 </template>
 
 <script>
 export default {
-  name: "TextField",
+  name: 'TextField',
   props: {
-    type: { type: String, default: "text" },
+    type: { type: String, default: 'text' },
     label: { type: String, required: true },
     placeholder: { type: String, required: true },
     value: { type: String, required: true }
+  },
+  mounted() {
+    this.textType = this.type
+  },
+  data() {
+    return {
+      textType: 'password',
```

Comment:
Can you try this? If it works you can remove the `mounted` function

```suggestion
      textType: this.type,
```

Reply:
it works and removes` mounted`